pkg/alertmanager/alertmanager.go

```diff
@@ -92,6 +92,7 @@ type Config struct {
 	Retention                         time.Duration
 	MaxConcurrentGetRequestsPerTenant int
 	ExternalURL                       *url.URL
+	TmplExternalURL                   *url.URL
```

Comment:
This new field is not "config" in the same sense as the other fields in the struct, i.e. global config taken from a YAML file and applied to all Alertmanagers.

I think the `Alertmanager` struct would be a better fit for this field (you could add `tmplExternalURL` [here](https://github.com/grafana/mimir/blob/369932771fb4eacef7209b4fd5046965af999513/pkg/alertmanager/alertmanager.go#L130-L131)).

Reply:
Got it! Changed it now 👍